### PR TITLE
Updated tooling dependencies on System.Web.*  to latest versions (#148)

### PR DESCRIPTION
**Pull request by svrx**
As discussed in #148, I've updated the tooling dependencies of core v3 of to latest versions.

### CODE REVIEW

File: RazorGenerator.Core.v3/RazorGenerator.Core.v3.csproj [hunk 1]
```diff
@@ -41,21 +41,23 @@
...
+    <Reference Include="System.Web.Helpers, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.AspNet.WebPages.3.2.3\lib\net45\System.Web.Helpers.dll</HintPath>
+    </Reference>
+    <Reference Include="System.Web.Mvc, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.AspNet.Mvc.5.2.3\lib\net45\System.Web.Mvc.dll</HintPath>
     </Reference>
     <Reference Include="System.Web.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Microsoft.AspNet.Razor.3.0.0\lib\net45\System.Web.Razor.dll</HintPath>
+      <HintPath>..\packages\Microsoft.AspNet.Razor.3.2.3\lib\net45\System.Web.Razor.dll</HintPath>
```
Comment (pranavkm) [inline RazorGenerator.Core.v3/RazorGenerator.Core.v3.csproj]: Does updating to this version fix the issue you were seeing in your application?

Comment (svrx) [inline RazorGenerator.Core.v3/RazorGenerator.Core.v3.csproj]: Tested only in development, for now, and results are positive.
Essentially the problem was occurring in a view section with inline JS inside, in which the parser was mixing up the closing brace (}) of a script with the closing brace of the section leading to problems.
When running the JITed version all was fine, and the only difference was that different version of compiler were being used.

**Comment by pranavkm** [comment]
Could you also bump up the version number for the package - https://github.com/RazorGenerator/RazorGenerator/blob/master/CommonAssemblyInfo.cs#L7 - to 2.5.0?

**Comment by svrx** [comment]
Done. :D

**Comment by pranavkm** [comment]
Thanks for the PR!

**Pull request merged**